common/archiver/gcloud/connector/client_test.go

```diff
@@ -220,8 +219,8 @@ func (s *clientSuite) TestGet() {
 
 func (s *clientSuite) TestWrongGoogleCredentialsPath() {
 	ctx := context.Background()
-	os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", "/Wrong/path")
-	_, err := connector.NewClient(ctx, &config.GstorageArchiver{})
+	s.T().Setenv("GOOGLE_APPLICATION_CREDENTIALS", "/Wrong/path")
```

Comment:
TIL

Reply:
yea - I'm half glad they added it (yay auto-restoring!) and half "ugh why encourage env vars, they should be painful!"